src/components/ChatView.tsx

```diff
@@ -112,7 +112,7 @@ export const ChatView = ({
   const hasContent = isLoading || messageHistory.length > 0;
 
   const onSendQuery = useCallback(() => {
-    if (isLoading) return;
+    if (isLoading || query.trim().length === 0) return;
```

Comment:
We can extract out a variable to indicate the coupling between the disabled appearance and the disabled behavior:

```
const isRequestDisabled = isLoading || query.trim().length === 0;
```

Is the `useCallback` saving any render cycles here?

Reply:
Good callout! `onSendQuery` used to be a bit more complex, justifying the `useCallback`- but it doesn't need it anymore. 

Addressed these comments here: [a66a7f6](https://github.com/vectara/react-chatbot/pull/21/commits/a66a7f67d9cb6405eeeae938f0bd453e8cf38429)